README.md

```diff
@@ -37,8 +37,10 @@ This repository uses chart specific labels to tag versions of each chart. The la
 Each helm chart can be packaged separately and published to a container registry.
 
 ```shell
+# Build helm dependencies for a single helm-chart product
+$ helm dependency build ./charts/<helm-chart-path> --repository-config helm-repositories.yaml
 # Package a single helm-chart product
-helm package -d ./build ./charts/<helm-chart-name>
+helm package -d ./build ./charts/<helm-chart-path>
```

Comment:
nit: Lets add the `$` to this and the command below since we have some inconsistency in how they are written

Reply:
fixed